tools/zod2md-jsdocs/src/nx-plugin.cjs

```diff
@@ -0,0 +1,84 @@
+const path = require('node:path');
+
+const ZOD2MD_CONFIG_FILE = 'zod2md.config.ts';
+const TS_PATCH_TARGET_NAME = 'ts-patch';
+const GENERATE_DOCS_TARGET_NAME = 'generate-docs';
+
+/**
+ * Creates the ts-patch target configuration
+ * @returns {object} ts-patch target configuration
+ */
+const createTsPatchTargetConfig = {
+  command: 'ts-patch install',
+  cache: true,
+  inputs: ['sharedGlobals', { runtime: 'ts-patch check' }],
+};
```

Comment:
I would move this target into `tools/zod2md-jsdocs/project.json` - see [previous comment](https://github.com/code-pushup/cli/pull/1162/changes#r2610811720).